### PR TITLE
Add fetchFromIssuer with ConnectionBuilder parameter

### PR DESCRIPTION
**Pull request by xuhongxu96**
Add `fetchFromIssuer` with `ConnectionBuilder` parameter for `AuthorizationServiceConfiguration`.

### CODE REVIEW

File: library/java/net/openid/appauth/AuthorizationServiceConfiguration.java [hunk 1]
```diff
@@ -224,6 +224,28 @@ public static void fetchFromIssuer(@NonNull Uri openIdConnectIssuerUri,
         fetchFromUrl(buildConfigurationUriFromIssuer(openIdConnectIssuerUri), callback);
     }
 
+    /**
+     * Fetch an AuthorizationServiceConfiguration from an OpenID Connect issuer URI, using
```
Comment (Hmawn) [inline library/java/net/openid/appauth/AuthorizationServiceConfiguration.java:228]: []()```
- # _**@**_``
```SGNrftalo

Comment (Hmawn) [reply library/java/net/openid/appauth/AuthorizationServiceConfiguration.java:228]: #131

**Review by Hmawn — APPROVED** [review]
Go

File: library/java/net/openid/appauth/AuthorizationServiceConfiguration.java (hunk 1 above)
Comment (xuhongxu96) [reply library/java/net/openid/appauth/AuthorizationServiceConfiguration.java:228]: Sorry I'm not sure if you are writing meaningful words

**Comment by robertschmid** [comment]
Please accept this merge request. I've had to replicate the same solution in my own code.

**Comment by agologan** [comment]
LGTM

**Comment by cumcum8197** [comment]
01c23545591d39e4a50878b0f95ff8cd8c3a9e44

**Pull request merged**